Bindle.Tests/Integration.cs

```diff
@@ -44,8 +181,6 @@ public async Task CanFetchYankedInvoices()
         [Fact]
         public async Task CanCreateInvoices()
         {
-            // TODO: this results in creating the invoice, so it can't be run twice!
```

Comment:
Thank you SOOOOOOOOOOOO MUCH for addressing this stuff.  It was a poor state for me to leave the tests in and I really appreciate you sorting it out.

Reply:
No, this is fantastic, it made it pretty straightforward to add bindle to wagi-dotnet so thank you !!